DatadogCrashReporting/Sources/PLCrashReporterIntegration/PLCrashReporterIntegration.swift

```diff
@@ -57,4 +57,21 @@ internal final class PLCrashReporterIntegration: ThirdPartyCrashReporter {
     func purgePendingCrashReport() throws {
         try crashReporter.purgePendingCrashReportAndReturnError()
     }
+
+    func generateBacktrace() throws -> BacktraceReport {
+        let liveReportData = try crashReporter.generateLiveReportAndReturnError()
+        let liveReport = try PLCrashReport(data: liveReportData)
+
+        // This is quite opportunistic - we map PLCR's live report through existing `DDCrashReport` builder to
+        // then extract essential elements for assembling `BacktraceReport`. It works for now, but be careful
+        // with how this evolves. We may need a dedicated `BacktraceReport` builder that only shares some code
+        // with `DDCrashReport` builder.
```

Comment:
I guess it's mostly affected by the PLCR update?

Do we expect to have failed unit tests when something like this happen, or we should have some extra measures?

Reply:
This is PLCR-safe 👍. Basically, both for Crash Reports and Backtrace Reports, the value returned by PLCR is `PLCrashReport`. We then map it through our `DDCrashReportBuilder`, whereas implementing distinct `BacktraceReportBuilder` would be more correct and scalable. We don't need it today, hence I'm leaving this decision context in comment.